evm/Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-plonky2 = { path = "../plonky2" }
+plonky2 = { path = "../plonky2", features = ["timing"] }
```

Comment:
Are these needed? I thought the feature was enabled by default.

Reply:
It is enabled by default, so no they're not needed.